data/World/America/USA/mncors.json

```diff
@@ -0,0 +1,54 @@
+{
+    "name": "mncors.dot.state.mn.us",
+    "description": "Minnesota DOT CORS",
+    "urls": [
+        "http://mncors.dot.state.mn.us:2101"
+    ],
+    "reference": {
+        "url": "https://www.dot.state.mn.us/surveying/cors/mncors_faq.html",
+        "comments": "Search for 'datum' on the page to find the table of mountpoints."
+    },
+    "last_update": "2025-03-26",
+    "streams": [
+        {
+            "filter": {
```

Comment:
I've seen in their web page that they also suggest different geoids (although not mention to the vertical CRS is made).  @javier-jimenez-shaw-pix4d, for the next version maybe we need to discuss how to include information relative to the vertical system and which geoid model to use to get there (it would help in the case of the LN02 mount point in Switzerland).